qanta/wikipedia/cached_wikipedia.py

```diff
@@ -145,6 +145,9 @@ def __getitem__(self, key: str):
             except ImportError:
                 print("Error importing %s" % key)
                 page = None
+            except ValueError:
+                print("Error importing %s" % key)
+                page = None
```

Comment:
Ya, I'm not particularly happy with the error handling for this. Do you by chance have the stack trace so I have a sense of what kind of ValueError it is catching? This is probably good to add in any case